dr_scaffold/generators.py

```diff
@@ -36,12 +36,14 @@ class BaseGenerator:
     fields: Tuple[str]
     core_dir: str
     api_dir: str
+    is_full: bool
 
-    def __init__(self, app_name, model_name, fields):
+    def __init__(self, app_name, model_name, fields, is_full):
```

Comment:
- it's better to add this new argument as optional, usually it's a good idea to do so, since anyone upgrading from the previous version will have breaking changes.
- `is_full` does not really make it clear what it's about, i think it should be more explicit

Reply:
@elmkarami is_full  is defaulted to false when it's not passed as an option in CLI so it won't break old versions, and yes I agree naming is not the best we gotta find something that's self-explanatory.